ui/src/components/TeamModal.vue

```diff
@@ -189,7 +197,8 @@ export default {
     async delete(item) {
       this.closeDialog();
       try {
-        const response = await this.deleteTeam(item.name, this.organization);
+        const organization = this.isUnaffiliated ? null : this.parent;
```

Comment:
Shouldn't it be better to have `this.isAffiliated`? The logic might be clear but I'm not sure if we use `isUnaffiliated` somewhere else?

Reply:
I've changed it to `isGroup`, which is the variable used in the other components.